### PR TITLE
Rewrite drawing summary in ES6

### LINKED ISSUE #1224: When you have a drawing task with a question task associated after it, the summary only reports the drawing part
Working on test project as a demo and the task is to draw a circle on the image and then answer a multiple choice question associated with that marking, the summary after the classification is submitted only lists the information about the marker

screen shot 2015-07-25 at 3 05 20 pm

screen shot 2015-07-25 at 3 17 29 pm

### PR DESCRIPTION
**Pull request by eatyourgreens**
Extract the rendering of individual tool annotations into their own components, for clarity.
Render details subtasks using the summary component for the matching task.
Remove the Less/More toggle (I didn't think it added much, but I can add it back in if it's needed.)

Staging branch URL: https://drawing-summary.pfe-preview.zooniverse.org

Fixes #1224.

Screenshot of the summary for I Fancy Cats, showing the cuteness rating in the drawing summary.

# Required Manual Testing

- [x] Does the non-logged in home page render correctly?
- [x] Does the logged in home page render correctly?
- [x] Does the projects page render correctly?
- [x] Can you load project home pages?
- [x] Can you load the classification page?
- [x] Can you submit a classification?
- [x] Does talk load correctly?
- [x] Can you post a talk comment?

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
This project has a question task set up on the big points drawing tool:
https://drawing-summary.pfe-preview.zooniverse.org/projects/eatyourgreens/-project-testing-ground/classify

The summary should be available in English and Italian.

**Comment by eatyourgreens** [comment]
Here's what this summary looks like on the dev classifier, where there are a mixture of different drawing tasks.

Screenshot showing subtask summaries for multiple choice questions, text inputs and the slider task.

File: app/classifier/tasks/drawing/summary.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Markdown } from 'markdownz';
+import SummaryAnswer from './summary-answer';
+
+class DrawingSummary extends React.Component {
+  static propTypes = {
```
Comment (simensta) [inline app/classifier/tasks/drawing/summary.jsx]: I would specify propTypes as `DrawingSummary.propTypes = {...}`. Is there a reason to use the static property instead?

Comment (eatyourgreens) [reply app/classifier/tasks/drawing/summary.jsx]: `static` is ES6 syntax for declaring static methods and properties. I don't think there's any difference between using the ES6 syntax or declaring static properties using `ClassName.property`. We added `static` to babel during the React 16 upgrade (535c0125b031ef8ef73ead6df1ffc404d8d4f3fc) so it's been used in some of the recent ES6 components. I think babel converts this to the old-style `DrawingSummary.propTypes` anyway.

**Pull request merged**